st2common/st2common/util/param.py

```diff
@@ -81,6 +82,13 @@ def _create_graph(action_context, config):
     '''
     G = nx.DiGraph()
     system_keyvalue_context = {SYSTEM_SCOPE: KeyValueLookup(scope=FULL_SYSTEM_SCOPE)}
+
+    # If both 'user' and 'api_user' are specified, this prioritize 'api_user'
+    user = action_context['user'] if 'user' in action_context else None
+    user = action_context['api_user'] if 'api_user' in action_context else user
+    if user:
+        system_keyvalue_context[USER_SCOPE] = UserKeyValueLookup(scope=FULL_USER_SCOPE, user=user)
```

Comment:
Hm, is this desired?

I'm fine with this approach, but we should document this behavior (something along the lines that ``st2kv.user.foo`` will only work for actions which have user context attached to and and are executed via the API or similar).

Reply:
Besides that, I'm fine with this change.

Nice work :+1:

Reply:
Thank you very much for your review. I got it. I'll do that soon!

Reply:
@userlocalhost Can you please also open st2docs PR which documents this behavior?